### ISSUE TITLE
Cannot set height of editor to 100%

### ISSUE THREAD

**Issue by varmentano** [froala/wysiwyg-editor#2249]
Use case:  Need a way to configure the Froala editor such that it expands to the size of its containing div.

##### Expected behavior.
There should be some way to make the editor always expand to the height of its containing div. I'd like to set the height to 100%.  Ideally this would make the entire editor fit its containing grid, including the toolbar.

##### Actual behavior.
The documentation indicates only numbers are expected for the "height" option.  I can input '100%', this behaves mostly as I'd hope except it leads to 2 new problems:
1) The element with 'fr-element' class does not expand to the size of the element with 'fr-wrapper' class, leading to unexpected blur events when clicking lower into the white-space of the editor.  
2) The toolbar is now overflowing.

You can calculate the height of the container div using jQuery or javascript and use that as the passed in height.  However this causes problems if your containing div gets resized and again you still have to account for the toolbar size.

##### Steps to reproduce the problem.
See https://codepen.io/vincentarmentano/pen/xXzmjE
For the blurring problem: 
1) Click where the placeholder text is so cursor starts blinking.
2) Click slightly below that but still within the editor window, you'll see a blur event gets triggered.

##### OS.
Mac OSX 10.11.6
##### Browser.
Chrome Version 61.0.3163.100

**Comment by stefanneculai** [comment]
@varmentano the editor height can be set only via the `height` option.

**Comment by samisin** [comment]
> @varmentano the editor height can be set only via the `height` option.

That's not really what he said. He asked for the possibility to add support for height 100%. I agree on this. Quite surprised it's not possible and now having to resort to calculating it with JavaScript instead.

**Comment by akashnigam010** [comment]
Is there a way to give dynamic (%) height instead of just having a fixed one?

**Comment by alexsmolin24** [comment]
`
.fr-box {
    display: flex;
    height: 100%;
    flex-direction: column;

    .fr-wrapper {
      flex: 1;

      .fr-element {
        min-height: 100%;
      }
    }
  }
`

**Comment by MadeByDouglas** [comment]
this should be a built in option IMO, thanks @alexsmolin24 for the quick fix

**Comment by stephenje** [comment]
The height option accepts calc and a viewport height. Here's an example of how get to 100% height minus 55px:

```
let editor = new FroalaEditor(
    '#editor', 
    { 
        height: 'calc(100vh - 55px)' 
    }
)
```

**Comment by AFDevMike** [comment]
> The height option accepts calc and a viewport height. Here's an example of how get to 100% height minus 55px:
> 
> ```
> let editor = new FroalaEditor(
>     '#editor', 
>     { 
>         height: 'calc(100vh - 55px)' 
>     }
> )
> ```

While `height: 'calc(100vh - 55px)' ` does indeed work:   It uses view height as opposed to %.    

I am unable to get any version of height to work when it includes the `%`.   
The following will not work:
```
height: 'calc(100% - 55px)' 
height: '100%' 
```

In my app, the css in [@alexsmolin24 answer](https://github.com/froala/wysiwyg-editor/issues/2249#issuecomment-557454168) gets overridden and doesn't work either.

**Comment by Lxxrxns** [comment]
The solution alexsmolin24 suggested could use this minor improvement:

`.fr-wrapper{
  cursor: text;
}`

And then something like (using jQuery): 

`
$('.fr-wrapper').click(function(){
  $('.fr-element').focus();
});
`

So that if you click on the wrapper, it will focus the are where you can type.

**Comment by btsiders** [comment]
2024 and still seemingly no built-in way to do this. @alexsmolin24's fix still works in Froala v4.2.1, but I added a little to get around the automatic `max-height` being set in the inline style and the silly second toolbar that I don't use.

```
&.fr-box {
	display: flex;
	height: 100%;
	flex-direction: column;

	.fr-wrapper {
		flex: 1;
		max-height: unset !important;
			.fr-element {
			min-height: 100%;
		}
	}

	.fr-second-toolbar {
		display: none;
	}
}
```

This works for me to fill an entire Ionic Framework (v8.2.5) modal window.
